src/components/Amalgamation/ResultingBusinessName.vue

```diff
@@ -186,6 +198,12 @@ export default class ResultingBusinessName extends Mixins(AmalgamationMixin, Nam
     this.updateResources()
   }
 
+  /** Displays fetch/validation error from CorrectName shared component. */
+  displayError (error: string): void {
+    this.nameRequestError = error || 'Unknown error. Please try again.'
+    this.nameRequestErrorDialog = true
```

Comment:
Currently, there is no code patch where error is falsy (empty/null), but I added the fallback on line 203 anyway.